services/galley/src/Galley/API/LegalHold.hs

```diff
@@ -204,10 +222,10 @@ approveDevice (zusr ::: tid ::: uid ::: connId ::: req ::: _) = do
   legalHoldAuthToken <- Client.getLegalHoldAuthToken uid mPassword
   LHService.confirmLegalHold clientId tid uid legalHoldAuthToken
   LegalHoldData.setUserLegalHoldStatus tid uid UserLegalHoldEnabled
-  -- TODO: send event at this point (see also:
-  -- https://github.com/wireapp/wire-server/pull/802#pullrequestreview-262280386)
-  pure empty
   where
+    -- TODO: send event at this point (see also:
```

Comment:
This comment should go one line up, to where we're returning from this function.

Reply:
ah, seems like ormolu pulls this into the `where`, so I have to put it two lines up to keep it from doing that. Or I add a `pure ()` below. :D